### ISSUE TITLE
Viewing Email doesn't work

### ISSUE THREAD

**Issue by Zynde** [cuba-platform/imap-addon#57]
Hi,

I'm trying out the latest version of this add-on and I cannot view emails. No errors are thrown and noting seems to appear in the logs it just sits there waiting. I cannot view either from the Demo screen or the IMAP Message Browser screen which doesn't open at all.

Very rarely will any new messages appear in the Demo screen.

I am using a Gmail account in my testing.

**Comment by smianna** [comment]
Hello @Zynde,
Thank you for your feedback! As far as we understand, you are using the imap-addon-demo project. 
Could you please provide more details about how you have set up the IMAP configuration? Have you tried recreating your IMAP configuration? Have you changed the default value of the *Custom flag* field in IMAP Configuration Editor?

**Comment by Zynde** [comment]
Hi, I'm not sure I can tell you much more as almost anything I do doesn't work.
I got the latest component and demo. Built them locally. The recent updates did allow me to create a config that connected. It was setup for a Gmail account. Apart from that I touched no other settings.

Only once did it pick up half a dozen messages otherwise it just does nothing on the screens. There appears to be nothing in the logs that I can see indicating an error.

![image](https://user-images.githubusercontent.com/31379998/41188730-6e2e2164-6c05-11e8-8b06-1a021b7703c8.png)

**Comment by smianna** [comment]
Hi @Zynde,
Please, upload the latest updates and try to re-create your database.
After that, when adding an IMAP configuration, you should specify a new custom flag (its name should differ from the default one).
Please, let us know if the issue persists after following the abovementioned steps.

**Comment by Zynde** [comment]
Hi @smianna 

I tried the latest version today and it still doesn't seem to work. I created a new database and setup a fresh configuration as you describe. It seemed to pick up 4 emails from hundreds but not in any particular order or date range. It then froze and I couldn't do anything. I restarted the app and managed to view one email (a first) then it all froze again. 

I don't know what it is doing but something is not happy.
